### ISSUE TITLE
Build release packages for linux/windows/macOS + add to homebrew package manager (macOS)

### ISSUE THREAD

**Issue by 0xdevalias** [mandiant/GoReSym#44]
The README says:

- https://github.com/mandiant/GoReSym#usage
  - > You can download pre-built `linux` and `windows` GoReSym binaries from the [Releases tab](https://github.com/mandiant/GoReSym/releases/).

Yet looking at the releases page, there doesn't seem to have been windows/linux binary releases since v2.4:

- https://github.com/mandiant/GoReSym/releases/
- https://github.com/mandiant/GoReSym/releases/tag/v2.4

And despite this issue implying this should be able to be built for macOS, I don't see any binary macOS releases:

- https://github.com/mandiant/GoReSym/issues/24

**Comment by 0xdevalias** [comment]
Actually, nevermind.. I just downloaded the `GoReSym.zip` from the latest release, and it seems like it contains all 3 binaries in the zip file:

- `GoReSym_lin`
- `GoReSym_mac`
- `GoReSym_win.exe`

This differs to what I would have expected (the 3 binaries to have been separately added to the release, as seems to be standard for a lot of projects), but I guess this works fine.

---

It might be worth updating the README to mention that macOS binaries are also built too, and possibly explicitly mentioning that all 3 are in the zip file.

**Comment by stevemk14ebr** [comment]
I've fixed the description, the recent versions of GoReSym have been able to include Mac binaries since we removed our cgo dependency on libyara.

I'm not willing to maintain third part package repositories unfortunately. If you'd like to take this up, it would be great however 🙂. I'll continue to publishes the releases as you see today.

The single zip is done because it compresses better and is less work to upload / download. I myself need both Linux and Windows releases so it's nice to have only a single download.
